chapters/ranker.rst

```diff
@@ -0,0 +1,155 @@
+==========================================
+A Guide to Jina Ranker
+==========================================
+
+.. meta::
+   :description: A guide on Jina Ranker
+   :keywords: Jina, Ranker
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+--------------------
+
+In a searching system, query accuracy is one of the most important aspects that users will concern. In most cases, we expect well-ordered query results to be presented.
+For one search request, we may have multiple matches and we want to have the most relevant match or probably top-K matches presented to users. On the other hand, we may also want to aggregate the top-K chunks to top-K matches.
+
+
+Before you start
+-------------------
+
+We expect you have a clean Python 3.7/3.8/3.9 (virtual) environment.
+Install Jina on your machine:
+
+.. highlight:: bash
+.. code-block:: bash
+
+    pip install -U jina
```

Comment:
Can you replace with? 

Requirements
-------------------
You have installed the latest stable release of Jina Core according to the instructions found `here
<https://docs.jina.ai/chapters/core/setup/index.html>`_.

Reply:
👍